solutions/silver/usaco-1182.mdx

```diff
@@ -16,10 +16,12 @@ it is never optimal to add more than once consecutively before dividing $a$ by
 to three operations in sum, we can instead first divide and then just add one,
 which has the same effect but requires only two operations.
 
-Then, if $a < b$, we can simulate our strategy above by allowing multiplication,
-division, and subtraction by one on $b$. We therefore either divide $b$ by two
-(and first subtract by one if $b$ is odd) or subtract $b$ $b - a$ times to
-directly reach $a$.
+Then, if $a < b$, as the optimality of the strategy above is already shown,
+let's reuse it and work on $b$ by allowing multiplication, division, and
+subtraction by one. We therefore either divide $b$ by two (and first subtract by
```

Comment:
doesn't the problem only allow you to work on a

Reply:
as stated, these two ops are equivalent
meaning what we do on b is exactly what we can do on a